modules/nodes-pods-autoscaling-creating-memory.adoc

```diff
@@ -94,7 +94,7 @@ spec:
       name: memory <9>
       target:
         type: AverageValue <10>
-        averageValue: 500Mi <11>
+        averageValue: 500m <11>
```

Comment:
This one we want to leave at `500Mi` (or 500 megabytes) since it's memory scaling, not CPU scaling.